src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -1109,6 +1126,58 @@ private void AddPatchNode(Patch patch, Vector2 position)
 
         patch.ApplyPatchEventVisuals(node);
 
+        var playerSpecies = patch.FindSpeciesByID(PlayerSpeciesID);
+        if (playerSpecies != null)
+        {
+            var playerPopulationIndicatorAmount = (int)Math.Ceiling(
```

Comment:
As this whole long block is inside this if, I'd recommend making this as a separate helper method.